cluster/examples/kubernetes/ceph/csi/rbd/storageclass-test.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: replicapool
   namespace: rook-ceph # namespace:cluster
 spec:
-  failureDomain: host
+  failureDomain: osd
```

Comment:
If it's replica 1 it shouldn't matter, right? But that's fine.

Reply:
I hope that it helps support the single node scenario slightly better if someone does use it in the world.